recipes/base_install.rb

```diff
@@ -217,15 +225,19 @@
 # Install NVIDIA and CUDA
 include_recipe "aws-parallelcluster::_nvidia_install"
 
-# Install FSx options
-include_recipe "aws-parallelcluster::_lustre_install"
+# FIXME:
+# # Install FSx options
+# include_recipe "aws-parallelcluster::_lustre_install"
 
-# Install EFA & Intel MPI
-include_recipe "aws-parallelcluster::efa_install"
-include_recipe "aws-parallelcluster::intel_mpi"
+# FIXME:
+# # Install EFA & Intel MPI
+# include_recipe "aws-parallelcluster::efa_install"
```

Comment:
You can enable `efa_install` and `intel_mpi` once https://github.com/aws/aws-parallelcluster-cookbook/pull/754 is merged.

Reply:
#754 is merged, enabled